src/plays/calendar/CalendarNavigation.tsx

```diff
@@ -0,0 +1,49 @@
+import { addMonths, format } from 'date-fns'
+import React from 'react'
+
+interface Props {
+  currentDate: Date,
+  onDateChange: (date: Date) => void
+}
+
+const CalendarNavigation = ({ currentDate, onDateChange }: Props) => {
+  const navigateTo = (direction: number) => {
+    /*
+      -1 = prev
+      0 = today
+      1 = next
+    */
+
+    if (direction === 0) {
+      onDateChange(new Date())
+      return
+    }
+
+    onDateChange(addMonths(currentDate, direction))
+  }
+
+  return (
+    <div className='vincentBCP-calendar-navigation'>
+      <button onClick={() => navigateTo(0)}>
+        Today
```

Comment:
`Today` button doesn't highlight today's date. You can use some CSS blinkers or similar highlights.